samples/snippets/create_table_schema_from_json.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pathlib
+
+
+def create_table(table_id: str):
+    orig_table_id = table_id
+    current_directory = pathlib.Path(__file__).parent
+    orig_schema_path = current_directory / "schema.json"
```

Comment:
I think our best bet for making the type checker happy is to turn this into a string.

```suggestion
    orig_schema_path = str(current_directory / "schema.json")
```